warehouse/query-core/src/main/java/datawave/query/jexl/visitors/JexlFormattedStringBuildingVisitor.java

```diff
@@ -305,6 +310,58 @@ public Object visit(ASTReferenceExpression node, Object data) {
         return sb;
     }
     
+    public static List<QueryMetric> formatMetrics(List<QueryMetric> metrics) {
+        List<QueryMetric> updatedMetrics = new ArrayList<QueryMetric>();
+        
+        // For each metric, update the query to be formatted (if applicable) and update
+        // the plan to be formatted
+        for (QueryMetric metric : metrics) {
+            JexlNode queryNode = null, planNode = null;
+            QueryMetric updatedMetric = new QueryMetric(metric);
+            String query = updatedMetric.getQuery();
+            String plan = updatedMetric.getPlan();
+            // If it is a JEXL query, set the query to be formatted
+            if (isJexlQuery(metric.getParameters())) {
+                try {
+                    queryNode = JexlASTHelper.parseJexlQuery(query);
+                    updatedMetric.setQuery(buildQuery(queryNode));
+                } catch (ParseException e) {
+                    log.error("Could not parse JEXL AST after performing transformations to run the query", e);
+                    
+                    for (String line : PrintingVisitor.formattedQueryStringList(queryNode)) {
+                        log.error(line);
+                    }
+                    log.error("");
+                    
+                    QueryException qe = new QueryException(DatawaveErrorCode.QUERY_EXECUTION_ERROR, e);
+                    throw new DatawaveFatalQueryException(qe);
+                }
+            }
+            // Format the plan (plan will always be a JEXL query)
+            try {
+                planNode = JexlASTHelper.parseJexlQuery(plan);
+                updatedMetric.setPlan(buildQuery(planNode));
+            } catch (ParseException e) {
+                log.error("Could not parse JEXL AST after performing transformations to run the query", e);
+                
+                for (String line : PrintingVisitor.formattedQueryStringList(planNode)) {
+                    log.error(line);
+                }
+                log.error("");
+                
+                QueryException qe = new QueryException(DatawaveErrorCode.QUERY_EXECUTION_ERROR, e);
```

Comment:
If we cannot parse the query, then lets leave the original query in the metric.  This should not cause an exception.